### ISSUE TITLE
Accessibility: screen readers don't detect if a user has activitypub permission enabled or disabled

### ISSUE THREAD

**Issue by talksina** [Automattic/wordpress-activitypub#791]
### Quick summary

A blind user can't detect if their user roles are or not enabled with ActivityPub usage.
After reaching the users management table, the last column should be "activitypub" and screen readers detect just a blank space because the condition, "enabled" or "disabled" activitypub permissions status, is represented just visually.

### Steps to reproduce

Pre-requirement: I'm using Windows Narrator screen reader but it applies to every existing assistive technology.
1. Windows user turn the screen reader on -ctrl+windows+enter-. Or command+f5 or preferences - acessibility -voiceover if running a mac.
2. with capslock+right arrow -or control+option+right arrow if running a mac-, reach the table and go on with the same combination till you hear the voice saying a blank space or at the best "emoji cross" and you'll see a black rectangle or similar form, indicating where screen reader focus is. You will notice that in every row, always moving forwards with capslock (or ctrl+option) + right arrow, every user has their activitypub blank without indication if it's enabled or not. It works then, but there is no way to know if they are enabled and who. 

### What you expected to happen

when navigating user table with a screen reader or tts software, focusing the "activitypub" column should read "activitypub enabled" if the user has activitypub permission, "activitypub disabled" if they are not allowed to use it.
It might be matter of ARIA roles or this stuff

### What actually happened

only feedback given on the activitypub column is a blank space. It's impossible for a blind user to understand if it's on or off.

### Impact

All

### Available workarounds?

No and the platform is unusable

**Comment by pfefferle** [comment]
Oh, good point! Sorry about that, we will fix that ASAP!

**Comment by talksina** [comment]
Found the problem, but I have not managed to solve it by myself. It's a quite simple HTML and text issue.
Here's the code for the interested column: 
[activitypub.txt](https://github.com/user-attachments/files/16045044/activitypub.txt)
there is just the emoji in place of the text "activitypub enabled" or "activitypub disabled"
screen readers intercept most emoji code but unluck wants not these ones
choose yourself if placing a text there, or a LABELED image with alternative text "activitypub allowed" and "activitypub disabled"
I tried fixing it myself, using the aria role (role="img" with aria-label="test") just to see if it worked but it doesn't - the role=img goes to misinterpret that one as a table cell and it creates more discomfort than help. Please fix asap as I am really in doubt whether I have enabled unwanted users!

**Comment by pfefferle** [comment]
I tried it with `<span title="Enabled">✓</span>` and the result on the Mac is quite good, because it always also reads the title of the column.
